### ISSUE TITLE
Container fails to start

### ISSUE THREAD

**Issue by oliverlabs** [Azure-Samples/ctfd-azure-paas#12]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ x ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> deploy the solution using the bicep file

### Any log messages given by the failure
> 2023-02-23T14:07:45.336Z ERROR - Container ctfd-app-robnd64thtq3u_1_445de949 for site ctfd-app-robnd64thtq3u did not start within expected time limit. Elapsed time = 230.680812 sec
2023-02-23T14:07:45.347Z ERROR - Container ctfd-app-robnd64thtq3u_1_445de949 didn't respond to HTTP pings on port: 8000, failing site start. See container logs for debugging.
2023-02-23T14:07:45.950Z INFO - Stopping site ctfd-app-robnd64thtq3u because it failed during startup.

### Expected/desired behavior
> The website does not load as expected giving a "504.0 GatewayTimeout" error

### OS and Version?
> Windows 11 Pro

### Versions
> latest

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by blacksud0** [comment]
Please share the build log with us so that we can determine the root cause of the problem more accurately.
Login to your Azure portal, Go to your `resource_group_name` > `ctfd-app-robnd64thtq3u` > `Development center` > Build log.

**Comment by oliverlabs** [comment]
[webapp_logs.zip](https://github.com/Azure-Samples/ctfd-azure-paas/files/10837638/webapp_logs.zip)
Hi Omer, @BlackSud0, please find the logs from the Deployment Centre attached.

**Comment by balteravishay** [comment]
thanks for reporting this issue @oliverlabs ! 
look like the database connection is not working as expected. did the deployment complete without any issues? if you go the app service's configuration blade, does it show green Vs next to the values that are hosted in the key-vault?
are you using the option to deploy the solution without vnet support or have you not changed that deployment parameter?

**Comment by oliverlabs** [comment]
Thanks Avishay, @balteravishay 
Yes, all 15 deployments succeeded successfully without any errors. Yes, the green Vs are shown next to 2 values (DATABASE_URL and REDIS_URL).
<img width="510" alt="ase" src="https://user-images.githubusercontent.com/70239916/221552811-91672b28-423c-4a01-b6d0-7605e861545d.png">

I checked the key vault and both values are present there. Could it be some firewall issue?

I deployed the solution with the VNet support (private endpoints for everything). I haven't changed any parameters.
